Dockerfile

```diff
@@ -494,6 +494,46 @@ RUN \
 
 # =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
+FROM sdk-go as sdk-govc
+
+USER root
+RUN \
+  mkdir -p /usr/libexec/tools /usr/share/licenses/govmomi && \
+  chown -R builder:builder /usr/libexec/tools /usr/share/licenses/govmomi
+
+ARG GOVMOMIVER="0.26.0"
+ARG GOVMOMISHORTCOMMIT="34586b6"
+
+USER builder
+WORKDIR ${GOPATH}/src/github.com/vmware/govmomi
+COPY ./hashes/govmomi /home/builder/hashes
+RUN \
+  sdk-fetch /home/builder/hashes && \
+  tar --strip-components=1 -xf govmomi-${GOVMOMIVER}.tar.gz && \
+  rm govmomi-${GOVMOMIVER}.tar.gz
+
+COPY --chown=0:0 --from=sdk-license-scan /usr/libexec/tools/ /usr/libexec/tools/
+COPY --chown=0:0 --from=sdk-license-scan /home/builder/license-scan/clarify.toml .
+RUN \
+  cp -p LICENSE.txt /usr/share/licenses/govmomi && \
+  go mod vendor && \
+  /usr/libexec/tools/bottlerocket-license-scan \
+    --clarify clarify.toml \
+    --spdx-data /usr/libexec/tools/spdx-data \
+    --out-dir /usr/share/licenses/govmomi \
+    go-vendor vendor
+
+RUN \
+  export CGO_ENABLED=0 ; \
+  export BUILD_VERSION_PKG="github.com/vmware/govmomi/govc/flags" ; \
+  go build -mod=vendor -o /usr/libexec/tools/govc -ldflags " \
+    -X ${BUILD_VERSION_PKG}.BuildVersion=${GOVMOMIVER} \
+    -X ${BUILD_VERSION_PKG}.BuildCommit=${GOVMOMISHORTCOMMIT} \
+    -X ${BUILD_VERSION_PKG}.BuildDate=$(date -u +"%Y-%m-%dT%H:%M:%SZ") \
```

Comment:
It's worth noting that using a moving date (time of build when not cached) will make creating a reproducible-build of this executable a challenge. Can the date instead be made into a static argument alongside the `GOVMOMIVER` and `GOVMOMISHORTCOMMIT` like `GOVMOMIDATE`? The value given could either be the release's timestamp or the timestamp that the sources are updated by a human.

Reply:
Others might disagree, but to me the build date should be the date that the binary is actually built. It's absolutely not reproducible, but I think that's kind of the point. It's signifying that it isn't the exact binary that was built last week, or the week before (although hopefully it is functionally the same!). I don't think this should confuse anyone since it's not even visible with `govc version`. You can only see the date when you run `govc version -l`.

Reply:
I see the build date giving users significantly less clarity on the artifact's origin in comparison to a timestamp related to a build's source revision. And, to me, the build date - specifically without a source-related date - seems like a misleading reference for users. It is very easy to mistake a recent date in version data meaning that its the product of a more recent release being consumed. That is, if the source revision is unchanged but the build date is, then users can easily (when looking for it) find a recent looking date but actually with a potentially much older version of the tool. I'd much rather see the "capture date" of when we updated the recipe or the source revision's original timestamp used over the arbitrary build timestamp.

If the date string isn't easily discoverable (and unlikely to be accessed) then perhaps we ought to omit it rather than offer a potentially confusing timestamp.

(in my view, this point of contention is analogous to `#!/bin/bash` v. `#!/usr/bin/env bash` - there are subtle differences in some contexts and profound in [others, that don't have a `/bin/bash`](https://nixos.org)!)